__tests__/cli.test.ts

```diff
@@ -152,13 +152,9 @@ describe('CLI', () => {
         return getJourneyStart(cli.output());
       };
 
-      expect((await output()).payload).toMatchObject({
-        params: { url: 'non-dev' },
-      });
+      expect((await output()).payload).toMatchObject({});
```

Comment:
this defaults the purpose of this test itself. We want to know whether the passed parameters from CLI are being used in the journey, but this change effectively discards that check itself. 

I would change the test to use a custom reporter and look for params being used as it is.

Reply:
i have tried to update the tests but ended up doing lot more than desired, i hope it;s ok :D

Reply:
Fixed the tests in a different way, other than using params.